### ISSUE TITLE
Feature Request: Add a Legend Below the Heatmap

### ISSUE THREAD

**Issue by mokkiebear** [mokkiebear/heatmap-tracker#3]
I’d like to request a small enhancement: adding a legend below the heatmap. Having a legend to explain the meaning of the heatmap colors (e.g., intensity levels or activity counts) would make it easier to interpret the data at a glance, especially for new users.

**Comment by dxcore35** [comment]
- Please add it only optional
- Or make it hidden only to be reveal on clicking on some UI element or hover over

**Comment by mokkiebear** [comment]
@dxcore35 yes, it will be optional as all other functionality like separateMonths, weekNumber and so on
